src/main/java/no/difi/sdp/client2/internal/DigipostMessageSenderFacade.java

```diff
@@ -42,7 +43,7 @@ public DigipostMessageSenderFacade(final Databehandler databehandler, final Klie
         MessageSender.Builder messageSenderBuilder = MessageSender.create(klientKonfigurasjon.getMeldingsformidlerRoot(),
                 keyStoreInfo,
                 wsSecurityInterceptor,
-                EbmsAktoer.avsender(databehandler.organisasjonsnummer),
+                EbmsAktoer.avsender(Organisasjonsnummer.of(databehandler.organisasjonsnummer.getOrganisasjonsnummer())),
```

Comment:
Trenger du å lage en ny `Organisasjonsnummer`-instans her? På https://github.com/difi/sikker-digital-post-klient-java/pull/51/files#diff-9e5ecd427f01b99ae0acd2829b0669b8R31 dytter du bare inn `String`en. Regner med den lager er `Organisasjunsnummer` under the hood, men da slipper du i hvert fall unna med litt mindre kode akkurat her :)